src/rai/rai/cli/rai_cli.py

```diff
@@ -96,7 +96,7 @@ def build_robot_identity():
             "as well as the robot's capabilities and limitations including sensor and actuator information. "
             "If there are any images provided, make sure to take them into account by thoroughly analyzing them. "
             "Your description should be thorough and detailed."
-            "Your reply should start with I am a ..."
+            "Your reply should start with You are ..."
```

Comment:
Just to double check, it's for identity building, but not for the identity itself, right? So the robot will still respond in the first person, like "I am a rosbot XL ..." to  "who are you question?".

(We really have to add tests to check that!)

Reply:
Yes, this is only for the system prompt

Reply:
Thanks! I'll test it tomorrow